custom-elements/reactions/HTMLInputElement.html

```diff
@@ -3,14 +3,247 @@
+  getParentElement, instance => setAttributes(instance, 'type', 'number'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'minLength', 'minlength', '2',
+  '3', 'minLength on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectBooleanAttributeWithDependentAttributes(
+  'multiple', 'multiple', 'multiple on HTMLInputElement',
+  'input', getParentElement, instance => setAttributes(instance, 'type', 'email'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'name', 'name', 'name1',
+  'name2', 'name on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'pattern', 'pattern', '[A-Za-z]{3}',
+  '[A-Za-z]{2}', 'pattern on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'placeholder', 'placeholder', 'value1',
+  'value2', 'placeholder on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectBooleanAttributeWithDependentAttributes(
+  'readOnly', 'readonly', 'readOnly on HTMLInputElement',
+  'input', getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectBooleanAttributeWithDependentAttributes(
+  'required', 'required', 'required on HTMLInputElement',
+  'input', getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'size', 'size', '1',
+  '2', 'size on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'src', 'src', '/custom1',
+  '/custom2', 'src on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributesToTestSrc(instance),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'step', 'step', '2',
+  '3', 'step on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'number'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithParentNode(
+  'type', 'type', 'text',
+  'number', 'type on HTMLInputElement', 'input',
+  getParentElement, HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'defaultValue', 'value', 'value1',
+  'value2', 'defaultValue on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
+
+testReflectAttributeWithDependentAttributes(
+  'value', 'value', 'value1',
+  'value2', 'value on HTMLInputElement', 'input',
+  getParentElement, instance => setAttributes(instance, 'type', 'text'),
+  HTMLInputElement
+);
```

Comment:
The value setter does not change the value content attribute, so this test is incorrect.